README.md

```diff
@@ -261,19 +239,39 @@ You can configure what events will be captured by passing the following argument
 
 For example:
 ```python
-from scenario import Context, Event, State
+import scenario
 
 def test_emitted_full():
-    ctx = Context(
+    ctx = scenario.Context(
         MyCharm,
         capture_deferred_events=True,
         capture_framework_events=True,
     )
-    ctx.run("start", State(deferred=[Event("update-status").deferred(MyCharm._foo)]))
+    in_state = scenario.State(
+        deferred=[
+            defer(MyCharm.on.config_changed, MyCharm._on_config_changed)  # No second arg could be all observers?
+        ]
+    )
+    in_state = scenario.State(
+        deferred = [
+            Deferred(MyCharm.on.config_changed, MyCharm._on_config_changed)  # No second arg could be all observers?
```

Comment:
I think I like this option the best, though I'd suggest `scenario.DeferredEvent` to be explicit.

In any case, let's discuss this today. What does "all observes" mean, and where does the info in the `event` object of the deferred event come from?